core/lib/spree/core/controller_helpers/auth.rb

```diff
@@ -83,7 +72,11 @@ def try_spree_current_user
           end
         end
 
-        deprecate try_spree_current_user: :spree_current_user, deprecator: Spree::Deprecation
+        deprecate \
+          try_spree_current_user: :spree_current_user,
+          redirect_back_or_default: 'Please use `redirect_to stored_spree_user_location_or(...)` when using solidus_auth_devise.',
```

Comment:
We're losing the information when not using solidus_auth_devise, right?

Reply:
That information was bad, we wrote it (me and @cpfergus1) before having a clear picture of all the ramifications.

I think not having auth-devise in the store is really niche and probably something that required custom implementations anyway. What do you think about having an example implementation in the upgrading instructions?

Reply:
That would be great, but I'm not very familiar with it. How would it look?